use/DifferentiationSNP.Rmd

```diff
@@ -120,22 +117,22 @@ genet.dist(Mydata1, method="WC84")
 ## Unsupervised clustering  
 
 We don't know the populations and we are looking for. As recommended by T.
-Jombart, with the function `gpr` we used the maximum  possible number of PCA
+Jombart, with the function `find.clusters()` we used the maximum  possible number of PCA
 axis which is 20 here. See detailed tutorial on this method for more information
 (http://adegenet.r-forge.r-project.org/) In this example, we used
```

Comment:
I'm going to make a direct link to the tutorial on GitHub. These are the ones that are up to date.